webapp/src/main/webapp/style.css

```diff
@@ -27,6 +27,35 @@ td, th {
 .warning {
   background: #ff7003;
   color: white;
+  
+}
+.content {
+  padding: 12px;
+}
+
+.error span, .warning span{
+  display: inline;
+}
+.error:not(.collapsed)  span, .warning:not(.collapsed)  span { 
+  display: none;
+}
+.error.collapsed p, .warning.collapsed p{
```

Comment:
Should there be a line between this and the previous section.

Reply:
yes i will add one to be consistent